vowel_duplicates.py

```diff
@@ -2,17 +2,17 @@
 def vowels(word):
     new_word = []
     vowel = ['a', 'e', 'i', 'o', 'u']
+    #get number of duplicates using set function
+    duplicates = len(word) - len(set(word))
+
     for x in word:
```

Comment:
it would be better if you loop through the set an not the word

Reply:
Alright, I have done it. You may look through please